### PR TITLE
craype fixes for scr

### PR DESCRIPTION
**Pull request by lukebroskop**
turn off static linking, add -ldl flag

### CODE REVIEW

**Review by eugeneswalker — DISMISSED** [review]

File: var/spack/repos/builtin/packages/scr/package.py [hunk 1]
```diff
@@ -123,6 +123,14 @@ class Scr(CMakePackage):
     variant('cntl_base', default=platform_tmp_default,
             description='Compile time default location for control directory.')
 
+    def flag_handler(self, name, flags):
+        if '%cce' in self.spec:
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/scr/package.py]:
```suggestion
        if self.spec.satisfies('%cce'):
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]

**Comment by tldahlgren** [comment]
@gonsie @CamStan Did you want to comment on this PR?

**Review by tldahlgren — APPROVED** [review]
LGTM.

**Comment by tldahlgren** [comment]
Would prefer to hold off on merging until Elsa and or Cameron weigh in on the changes.

File: var/spack/repos/builtin/packages/scr/package.py [hunk 2]
```diff
@@ -123,6 +123,14 @@ class Scr(CMakePackage):
     variant('cntl_base', default=platform_tmp_default,
             description='Compile time default location for control directory.')
 
+    def flag_handler(self, name, flags):
+        if self.spec.satisfies('%cce'):
+            if name in ['cflags', 'cxxflags', 'cppflags']:
+                return (None, flags, None)
+            elif name == 'ldflags':
+                flags.append('-ldl')
+            return (flags, None, None)
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/scr/package.py:132]:
```suggestion
        return (flags, None, None)
```

Comment (tldahlgren) [inline var/spack/repos/builtin/packages/scr/package.py:132]:
```suggestion
        return (flags, None, None)
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/scr/package.py (hunk 2 above)
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/scr/package.py]:
```suggestion
        return (flags, None, None)
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]

**Review by CamStan — DISMISSED** [review]
We're good with this once the suggested change is in.

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**